feature-libs/asm/components/customer-selection/customer-selection.component.html

```diff
@@ -1,83 +1,163 @@
-    <cx-form-errors
-      *cxFeature="'formErrorsDescriptiveMessages'"
-      [translationParams]="{
-        label: 'asm.customerSearch.searchTerm.label' | cxTranslate
-      }"
-      [control]="customerSelectionForm.get('searchTerm')"
-    ></cx-form-errors>
+    <div *ngIf="searchResults | async as results" #resultList>
+      <div
+        *ngIf="
+          isNoResultMessageInfoVisible(results, searchByCustomer, searchTerm)
+        "
+        class="cx-message cx-message-info selection-message-info"
+      >
+        <div class="cx-message-content">
+          <div class="cx-message-header">
+            <span class="cx-message-icon">
+              <cx-icon
+                class="cx-icon fas fa-info-circle"
+                ng-reflect-type="INFO"
+              ></cx-icon>
+            </span>
+            <span class="cx-message-text"
+              >{{ 'asm.customerSearch.noCustomerMatchResult' | cxTranslate }}
+            </span>
+            <span
+              #createCustomerLink
+              class="linkStyleLabel"
+              (click)="createCustomer()"
+            >
+              {{ 'asm.customerSearch.createCustomer' | cxTranslate }}
+            </span>
+          </div>
+        </div>
+      </div>
+      <div
+        *ngIf="isSearchResultsVisible(results, searchByCustomer)"
+        class="asm-results"
+      >
+        <button
+          #searchResultItem
+          *ngFor="let result of results.entries; let i = index"
+          [tabindex]="activeFocusedButtonIndex === i ? 0 : -1"
+          [attr.aria-selected]="activeFocusedButtonIndex === i"
+          [class.active]="activeFocusedButtonIndex === i"
+          (keydown.arrowup)="focusPreviousChild($event)"
+          (keydown.arrowdown)="focusNextChild($event)"
+          (keydown.arrowright)="focusInputText($event)"
+          (keydown.arrowleft)="focusInputText($event)"
+          (keydown.enter)="selectCustomerFromList($event, result)"
+          (keydown.escape)="closeResults($event)"
+          (keydown.home)="focusInputText($event)"
+          (keydown.end)="focusInputText($event)"
+          (click)="selectCustomerFromList($event, result)"
+        >
+          <div class="result-name">{{ result.name }}</div>
+          <div class="result-id">
+            <span>{{ result.uid }}</span>
+          </div>
+        </button>
+      </div>
+    </div>
 
-    <cx-form-errors
-      *cxFeature="'!formErrorsDescriptiveMessages'"
-      [control]="customerSelectionForm.get('searchTerm')"
-    ></cx-form-errors>
+    <div class="asm-results" *ngIf="isLoading && searchByCustomer">
+      <cx-dot-spinner
+        aria-hidden="false"
+        [attr.aria-label]="'common.loading' | cxTranslate"
+      ></cx-dot-spinner>
+    </div>
   </label>
-  <button type="submit" [class.active]="selectedCustomer">
-    {{ 'asm.customerSearch.submit' | cxTranslate }}
-  </button>
-</form>
+  <span class="searchLabel">{{
```

Comment:
This is a break change, if the feature is deliveried to minor release please handle it appropriately